src/document-fetcher.js

```diff
@@ -0,0 +1,145 @@
+/**
```

Comment:
Can we drop this file until the document fetch PR?

Reply:
sure